shared/actions/config.js

```diff
@@ -122,10 +122,10 @@ const bootstrap = (opts: $PropertyType<ConfigGen.BootstrapPayload, 'payload'>):
       logger.info('[bootstrap] bootstrapping on connect')
       dispatch(ConfigGen.createBootstrap({}))
     })
-    dispatch(registerListeners())
   } else {
     logger.info('[bootstrap] performing bootstrap...')
     Promise.all([
+      dispatch(registerListeners()),
```

Comment:
we need to make sure other platforms aren't hosed by this, maybe do a test build of osx

Reply:
will do, good idea